timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -391,3 +415,343 @@ def _get_model_type(self):
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        from statsforecast.models import CrostonOptimized
+
+        return CrostonOptimized
+
+
+class CrostonClassicModel(AbstractStatsForecastIntermittentDemandModel):
+    """Intermittent demand forecasting model using Croston's model where the smoothing parameter
+    is fixed to 0.1 [Croston1972]_.
+
+    Based on `statsforecast.models.CrostonClassic <https://nixtla.mintlify.app/statsforecast/docs/models/crostonclassic.html>`_.
+
+
+    References
+    ----------
+    .. [Croston1972] Croston, John D. "Forecasting and stock control for intermittent demands." Journal of
+        the Operational Research Society 23.3 (1972): 289-303.
+
+
+    Other Parameters
+    ----------------
+    n_jobs : int or float, default = 0.5
+        Number of CPU cores used to fit the models in parallel.
+        When set to a float between 0.0 and 1.0, that fraction of available CPU cores is used.
+        When set to a positive integer, that many cores are used.
+        When set to -1, all CPU cores are used.
+    max_ts_length : int, default = 2500
+        If not None, only the last ``max_ts_length`` time steps of each time series will be used to train the model.
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        from statsforecast.models import CrostonClassic
+
+        return CrostonClassic
+
+
+class IMAPAModel(AbstractStatsForecastIntermittentDemandModel):
+    """Intermittent demand forecasting model using the Intermittent Multiple Aggregation Prediction Algorithm
+    [Petropoulos2015]_.
+
+
+    Based on `statsforecast.models.IMAPA <https://nixtla.mintlify.app/statsforecast/docs/models/imapa.html>`_.
+
+
+    References
+    ----------
+    .. [Petropoulos2015] Petropoulos, Fotios, and Nikolaos Kourentzes. "Forecast combinations for intermittent
+        demand." Journal of the Operational Research Society 66.6 (2015): 914-924.
+
+
+    Other Parameters
+    ----------------
+    n_jobs : int or float, default = 0.5
+        Number of CPU cores used to fit the models in parallel.
+        When set to a float between 0.0 and 1.0, that fraction of available CPU cores is used.
+        When set to a positive integer, that many cores are used.
+        When set to -1, all CPU cores are used.
+    max_ts_length : int, default = 2500
+        If not None, only the last ``max_ts_length`` time steps of each time series will be used to train the model.
+        This significantly speeds up fitting and usually leads to no change in accuracy.
+    """
+
+    def _get_model_type(self):
+        from statsforecast.models import IMAPA
+
+        return IMAPA
+
+
+class ConformalizedZeroModel(AbstractStatsForecastIntermittentDemandModel):
```

Comment:
Maybe change the name to `ZeroModel`? We don't use the `Conformalized*` prefix for other models that are also based on conformalization, so maybe drop it here as well?

Reply:
I wanted to highlight that it's got a little more intellectual depth than just Zero, but fair enough :)